hera_mc/data/test_data/initialization_data_part_rosetta.csv

```diff
@@ -2,3 +2,7 @@ hpn,syspn,start_gpstime,stop_gptime
 SNPC000700,heraNode700Snap700,1275040818,
 SNPC000701,heraNode0Snap701,1275040818,
 SNPC000709,heraNode700Snap709,1275040818,
+SNPA000700,heraNode700Snap0,1230372018,
+SNPB000701,heraNode700Snap1,1230372018
```

Comment:
I'm not familiar with the csv format. Do we need a trailing comma here, like in the previous lines?

Reply:
Looks like no.